samples/net/sockets/civetweb_websocket_server/README.rst

```diff
@@ -0,0 +1,64 @@
+.. _sockets-civetweb-websocket-server-sample:
+
+Civetweb WebSocket Server sample
+################################
+
+Overview
+********
+
+This sample application uses the HTTP APIs provided by the external
+`Civetweb <https://github.com/civetweb/civetweb>`_ module to create an WebSocket
+server demonstrating selected Civetweb features.
+The Civetweb module is available as a west :ref:`module <modules>`.
+
+The source code for this sample application can be found at:
+:zephyr_file:`samples/net/sockets/civetweb_websocket_server`.
+
+Requirements
+************
+
+- A board with hardware networking
+- The Civetweb module (made available via west)
+
+Building and Running
+********************
+
+This sample was tested on the NUCLEO H745ZI-Q board, so this is the recommended target.
+
+Build it with:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/net/sockets/civetweb_websocket_server
+   :board: nucleo_h745zi_q_m7
+   :goals: build
+   :compact:
+
+The sample application uses a static IP configuration.
+
+After flashing the board, the server can be accessed with the web browser
+of your choice (preferably Chrome) under ``192.0.2.2`` IPv4 address.
```

Comment:
Zephyr networking samples have standardized on 192.0.2.1 being the **device** address, and 192.0.2.2 being the (peer) **host** address. Here you have it reversed, which will lead to no end of confusion. Please fix it.

Reply:
done

Reply:
Thanks for update.